guides/common/modules/proc_registering-capsule-to-satellite-server.adoc

```diff
@@ -16,7 +16,7 @@ For more information on manifests and repositories, see {ContentManagementDocURL
 
 .Proxy and Network Prerequisites
 * The {ProjectServer} base operating system must be able to resolve the host name of the {SmartProxy} base operating system and vice versa.
-* The base operating system on which you want to install {SmartProxyServer} must not be configured to use a proxy to connect to the Red Hat CDN.
+* There should be no proxy configured for the {SmartProxyServer} to connect to the {Project}.
```

Comment:
```suggestion
* Ensure there is no HTTP proxy required for {SmartProxyServer} to connect to {ProjectServer}.
```

Reply:
As I understand it, {ProjectServer} will resolve to "Capsule Server", is this correct? If this is true, then this would still be incorrect. I don't believe there is any part in the documentation that instructs the user to install the Capsule server by making a connection to the Red Hat CDN at any time. All OS updates to include Capsule installation packages come from the Satellite as a client registered to the Satellite. To ensure there are no issues an http proxy cannot be configured between the Satellite and Capsule server.

Reply:
No. Project Server resolves to Foreman Server, and SmartProxyServer resolves to Smart Proxy Server. See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc and https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc

Reply:
Like @taftsanders, I also disagree with `no HTTP proxy required` because it's not strong enough. HTTP proxies often interfere with HTTPS client certificates so it is advisable (in Satellite's case you could even say unsupported) to have a proxy in place.

Reply:
Yes, now I see it too: "Ensure" + "required" is a weird combination.